datafusion/sqllogictest/test_files/math.slt

```diff
@@ -713,9 +713,7 @@ EXPLAIN SELECT log(NULL, c2) from aggregate_simple;
 logical_plan
 01)Projection: Float64(NULL) AS log(NULL,aggregate_simple.c2)
 02)--TableScan: aggregate_simple projection=[]
-physical_plan
-01)ProjectionExec: expr=[NULL as log(NULL,aggregate_simple.c2)]
-02)--DataSourceExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/aggregate_simple.csv]]}, file_type=csv, has_header=true
+physical_plan DataSourceExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/aggregate_simple.csv]]}, projection=[NULL as log(NULL,aggregate_simple.c2)], file_type=csv, has_header=true
```

Comment:
and there you can see the projection expressions moved into DataSourceExec per the plan 🚀